npm_and_yarn/helpers/build

```diff
@@ -23,4 +23,4 @@ cp -r \
   "$install_dir"
 
 cd "$install_dir"
-npm ci --no-audit --fetch-timeout=600000 --fetch-retries=5 --no-dry-run --no-ignore-scripts
```

Comment:
- no audit is now enforced via global env var
- both `--no-ignore-scripts` and `--no-dry-run` are the default but were necessary here to workaround the global `.npmrc` file npm 6 hack... with the hack removed, we don't need to specify either